lms/djangoapps/onboarding/forms.py

```diff
@@ -194,17 +242,60 @@ def clean_language(self):
 
         raise forms.ValidationError(ugettext_noop('Please select language.'))
 
+    def clean_organization_name(self):
+        organization_name = self.cleaned_data['organization_name']
+
+        if not self.data.get('is_currently_employed') and not organization_name:
+            raise forms.ValidationError(ugettext_noop('Please enter Organization Name.'))
+
+        return organization_name
+
+    def clean_role_in_org(self):
+        if self.instance.organization:
+            if not self.cleaned_data['role_in_org']:
+                raise forms.ValidationError(ugettext_noop(NO_OPTION_SELECT_ERROR.format(
+                    'Role in the Organization')))
+        return self.cleaned_data['role_in_org']
+
     def clean_start_month_year(self):
         if self.instance.organization:
+            if not self.cleaned_data['start_month_year']:
+                raise forms.ValidationError(ugettext_noop("Please enter a valid start month/year"))
+
             start_month_year = datetime.strptime(
                 self.cleaned_data['start_month_year'],
                 '%m/%Y')
-
             if start_month_year > datetime.now():
                 raise forms.ValidationError(ugettext_noop("Please enter a valid start month/year"))
 
         return self.cleaned_data['start_month_year']
 
+    def clean_hours_per_week(self):
+        if self.instance.organization:
+            if not self.cleaned_data['hours_per_week']:
+                raise forms.ValidationError(
+                    ugettext_noop(EMPTY_FIELD_ERROR.format('Hours per week')))
+        return self.cleaned_data['hours_per_week']
+
+        return organization_name
```

Comment:
Are both of these return statements properly-indented and reachable?

Reply:
Thanks, Done